src/main/java/ua/lviv/iot/parkingServer/service/ParkingService.kt

```diff
@@ -1,51 +1,27 @@
 package ua.lviv.iot.parkingServer.service
 
-import jakarta.annotation.PostConstruct
-import jakarta.annotation.PreDestroy
-import java.util.concurrent.ConcurrentHashMap
-import java.util.concurrent.atomic.AtomicLong
 import org.springframework.stereotype.Service
 import ua.lviv.iot.parkingServer.annotation.TrackMetrics
-import ua.lviv.iot.parkingServer.datastorage.ParkingFileStore
+import ua.lviv.iot.parkingServer.repository.ParkingRepository
 import ua.lviv.iot.parkingServer.exception.EntityNotFoundException
 import ua.lviv.iot.parkingServer.model.Parking
 import ua.lviv.iot.parkingServer.service.interfaces.ParkingServiceInterface
 
 @TrackMetrics
 @Service
-class ParkingService(private val parkingFileStore: ParkingFileStore) : ParkingServiceInterface {
-    private val parking: MutableMap<Long, Parking> = ConcurrentHashMap()
-    private var id: AtomicLong = AtomicLong(1L)
+class ParkingService(
+    private val parkingRepository: ParkingRepository
+) : ParkingServiceInterface {
+    override fun findAllEntities(): List<Parking> = parkingRepository.findAll()
 
-    override fun findAllEntities(): List<Parking> = parking.values.toList()
+    override fun findEntityById(id: String): Parking = parkingRepository.findById(id)
+        .orElseThrow{ EntityNotFoundException("Parking with id=$id not found") }
```

Comment:
please run detekt and fix code style issues.

I see a small issue here

Reply:
Detekt can't see any issues

Reply:
1. try `command+option+L`.
2. try to make a line with more than 120 symbols or some another codestyle issue. Then run detekt. Does it reports any issue? It's possible that the plugin is not configured properly, so it doesn't report at all

What's wrong here:
`orElseThrow{` -> `orElseThrow {`

Reply:
✅